sdk/search/Azure.Search.Documents/samples/Sample04_FieldBuilder.md

```diff
@@ -0,0 +1,127 @@
+# Azure.Search.Documents Samples - FieldBuilder
+
+The `FieldBuilder` class allows you to define a Search index from a model type,
+though not all property types are supported by a Search index without workarounds.
+
+## Model
+
+Consider the following model, which declares a property of an `enum` type.
```

Comment:
Feels kind of weird to start the sample with workarounds.  Might be better to show normal use and include this at the end?  Or at least add an intro paragraph explaining how you're decorating the properties with the attributes and what they effectively mean before we explain the enum weirdness?

Reply:
Fair point. I put this in samples but originally it was meant to be just a workaround, but given it's location I agree at least rewording this would be good. The sample code itself should already serve the "just the basics" well enough.

Reply:
The more I think about this, the more I think changing the title (and link description in the README) to set the expectation this is a workaround is probably better. That's really what was intended, but I agree that just calling the sample "FieldBuilder" is misleading. We already have a sample in the package README, so I don't think another is necessary here.

@AlexGhiondea, thoughts?

Reply:
Refocused the sample (file names, link descriptions, title, etc.) on `[FieldBuilderIgnore]`.